core/types/transaction_marshalling.go

```diff
@@ -92,6 +93,29 @@ func (tx DynamicFeeTransaction) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&enc)
 }
 
+func (tx SignedBlobTx) MarshalJSON() ([]byte, error) {
```

Comment:
You need to add two new fields to txJSON for MaxFeePerDataGas and BlobVersionedHashes,  then appropriately marshal/unmarshal those values in your implementations.

Reply:
I updated